src/pages/home/report/ReportActionCompose.js

```diff
@@ -300,6 +299,14 @@ class ReportActionCompose extends React.Component {
     }
 }
 
+const CREATE_MENU_ITEMS = {
+    ATTACHMENT: {
+        icon: Paperclip,
+        text: 'Upload Photo',
+        onSelected: () => {},
+    },
+};
```

Comment:
I don't know where to put this, it doesn't feel right to have it at the top. Maybe in a separate file...

Reply:
Good point, but it's probably fine to match our existing usages for now.